dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationManager.java

```diff
@@ -137,6 +138,9 @@ protected void modified(ComponentContext compcontext, Map<String, Object> proper
         setUseJandex(useJandexValue == null ? false : useJandexValue);
         //System.setProperty("com.ibm.ws.jandex.enable", useJandexValue.toString()); // Temporary -- REMOVE THIS LATER ////
 
+        String annotationScanLibraryValue = getProperty(properties, "annotationScanLibrary", "");
```

Comment:
Shouldn't the changes in this entire class be undone?  `com.ibm.ws.app.manager/resources/OSGI-INF/metatype/metatype.xml` wasn't updated to add an `annotationScanLibrary` attribute so this will always be null converted to `""`.

Reply:
Perhaps. If I recall I followed the call chain which led me to adding stuff like this. I'll undo it and test it. If it works without it then I'll commit the revert.